.openshift-ci_cd/Jenkinsfile

```diff
@@ -23,4 +29,12 @@ node ('master') {
     sh 'oc delete route nodejs-ex-dev || true'  // Bugfix for https://github.com/openshift/origin/pull/9425 not getting into latest OpenShift release
     sh "oc process -f .openshift-ci_cd/deployment-template.yaml -v 'APP_NAME=nodejs-ex-dev,IMAGE_STREAM_NAMESPACE=${k8s_namespace},IMAGE_STREAM_TAG=nodejs-ex:latest' | oc apply -f -"
     sh 'oc annotate is nodejs-ex misc_$(date +%s)=foo || true'  // Hack for 0-2 minute pause caused by https://github.com/openshift/origin/issues/9018
+
+    stage 'Run Integration Tests'
+    sh 'npm config set prefix=$HOME/node && export PATH=$HOME/node/bin:$PATH && npm install -r package.json'
+    sh "IP=`oc get route nodejs-ex-dev -n ${k8s_namespace} -o go-template=\'{{.spec.host}}\'` node_modules/mocha/bin/mocha --timeout 5000 tests/app_test.js --reporter xunit | tee report.xml"
```

Comment:
This is not actually working at all. If I put there:
```bash
IP="nonexisting.xip.io" ./node_modules/mocha/bin/mocha --timeout 5000 tests/app_test.js --reporter xunit | tee report.xml
```
It still says success. It is weird that it does not fail when using xip.io.

If the app is not deployed at all it would still succeed.